docs/development.md

```diff
@@ -22,29 +22,29 @@ Docker compose config](docker-compose-dev.yml).
 We mount local checkouts of all the code into these containers, so first you'll
```

Comment:
I don't know why it's saying this file has changed, when I run the diff locally it doesn't show up :thinking: